// Error: break fuera de loop
function main() {
    let x: integer = 5;
    
    if (x > 0) {
        break;  // ERROR: break fuera de loop
    }
}
